lib/pure/collections/tables.nim

```diff
@@ -1206,18 +1206,17 @@ type
 
 # ------------------------------ helpers ---------------------------------
 
-proc rawGetKnownHC[A, B](t: OrderedTable[A, B], key: A, hc: Hash): int =
-  rawGetKnownHCImpl()
-
-proc rawGetDeep[A, B](t: OrderedTable[A, B], key: A, hc: var Hash): int {.inline.} =
-  rawGetDeepImpl()
-
-proc rawGet[A, B](t: OrderedTable[A, B], key: A, hc: var Hash): int =
-  rawGetImpl()
+template initImpl(t: OrderedTable, size: int) =
+  ## Initializes `t` to a valid empty state. Called internally by the
+  ## ``checkIfInitialized`` to set lazily set up the table before it is used
```

Comment:
```suggestion
  ## Initializes `t` to a valid empty state. Called internally by
  ## ``checkIfInitialized`` to lazily set up the table before it is used
```

Reply:
Thanks for reporting these. I tend to adjust documentation comments multiple times, and sometimes the remnants of previous iterations get past my proof-reading.